config/jobs/kubeflow/kubeflow-postsubmits.yaml

```diff
@@ -35,6 +35,8 @@ postsubmits:
     annotations:
       testgrid-dashboards: sig-big-data
       description: Postsubmit tests for Kubeflow.
+      testgrid-alert-email: kubeflow-engineering@google.com
```

Comment:
Maybe we should add a TODO to setup a public alerts mailing list so that other people can share the joy of being alerted to test failures.